platform/android/export/export.cpp

```diff
@@ -702,6 +718,36 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return OK;
 	}
 
+    static Error save_gradle_project_so(void *p_userdata, const SharedObject &p_so) {
```

Comment:
How much of this function was copied from somewhere else (if any) and how much can be refactored to reuse in multiple places?

Reply:
Almost all of it was copied from a function called "save_apk_so", the only thing that changed was the function that writes to a file. I can factor that out

Reply:
But, I don't know how to deal with function pointers in C. Also, save_gradle_project_so takes in different arguments and does less things than save_apk_so. I'm thinking that leaving it as it's own function might be a better idea.

Reply:
Sounds good, thanks.